src/Sparrow/Json/LazyCompressedStringValue.cs

```diff
@@ -126,5 +127,52 @@ public string Substring(int startIndex, int length)
         {
             return ToString().Substring(startIndex, length);
         }
+
```

Comment:
Missing equality to `LazyStringValue`

Reply:
Will address it in a different issue: https://issues.hibernatingrhinos.com/issue/RavenDB-12731